eos-copy-host-mali-driver.service

```diff
@@ -0,0 +1,21 @@
+# Copy the host GL driver for Mali to a place where Flatpak will mount it
+
+[Unit]
+Description=Copy Mali GL driver for Flatpak
+DefaultDependencies=no
+Conflicts=shutdown.target
+Wants=local-fs.target
+After=local-fs.target
+Before=multi-user.target
+
+# Only run on ARM systems with the Mali driver
+ConditionArchitecture=arm
+ConditionPathExists=/usr/lib/arm-linux-gnueabihf/mali400-egl-x11
+
```

Comment:
We can also add ConditionNeedsUpdate=/var so that this is only re-run after an OS upgrade. In this case I think we'd need to also do the copy in the image builder (or run the script in a chroot hook) as I'm not sure the update condition is true on first boot.